okta/komand_okta/triggers/users_added_removed_from_group/trigger.py

```diff
@@ -0,0 +1,101 @@
+import komand
+import time
+from .schema import UsersAddedRemovedFromGroupInput, UsersAddedRemovedFromGroupOutput, Input, Output, Component
+# Custom imports below
+import itertools
+from komand.exceptions import PluginException
+
+
+class UsersAddedRemovedFromGroup(komand.Trigger):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='users_added_removed_from_group',
+                description=Component.DESCRIPTION,
+                input=UsersAddedRemovedFromGroupInput(),
+                output=UsersAddedRemovedFromGroupOutput())
+
+    def run(self, params={}):
+        """Run the trigger"""
+        group_list = params.get(Input.GROUP_IDS)
+        okta_url = self.connection.okta_url
+        current_list = list()
+        group_names = list()
+        for group in group_list:
+            api = f"{okta_url}/api/v1/groups/{group}/users"
+            # Build a reference list to check for updates against
+            response = self.connection.session.get(api)
+            if response.status_code in range(400, 499):
+                raise PluginException(cause="Okta returned a 4xx status code",
```

Comment:
Can we link the specific code to an error message that Okta has available?

For all Exception lines in this PR

Reply:
@mhofert-r7 we should implement this. Okta has status codes available in their documentation.